### PR TITLE
BMC API domain whitelisted for pnap driver

### PR DESCRIPTION
**Pull request by pajuga**

### CODE REVIEW

File: pkg/data/management/machinedriver_data.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func addMachineDrivers(management *config.ManagementContext) error {
 	if err := addMachineDriver(PacketDriver, "https://github.com/packethost/docker-machine-driver-packet/releases/download/v0.2.2/docker-machine-driver-packet_linux-amd64.zip", "https://packethost.github.io/ui-driver-packet/1.0.2/component.js", "e03c6bc9406c811e03e9bc2c39f43e6cc8c02d1615bd0e0b8ee1b38be6fe201c", []string{"api.packet.net", "packethost.github.io"}, false, false, false, management); err != nil {
 		return err
 	}
-	if err := addMachineDriver(PhoenixNAPDriver, "https://github.com/phoenixnap/docker-machine-driver-pnap/releases/download/v0.1.0/docker-machine-driver-pnap_0.1.0_linux_amd64.zip", "", "5f25a7fbcaca0710b7290216464ca8433fa3d683b59d5e4e674bef2d0a3ff6c7", []string{"api.securedservers.com"}, false, false, false, management); err != nil {
+	if err := addMachineDriver(PhoenixNAPDriver, "https://github.com/phoenixnap/docker-machine-driver-pnap/releases/download/v0.1.0/docker-machine-driver-pnap_0.1.0_linux_amd64.zip", "", "5f25a7fbcaca0710b7290216464ca8433fa3d683b59d5e4e674bef2d0a3ff6c7", []string{"api.securedservers.com", "api.phoenixnap.com"}, false, false, false, management); err != nil {
```
Comment (dramich) [inline pkg/data/management/machinedriver_data.go:121]: Is api.securedservers.com still needed? It looks like securedservers.com 301s to `https://phoenixnap.com/servers/dedicated`

Comment (dramich) [reply pkg/data/management/machinedriver_data.go:121]: Running `curl -Lv https://api.securedservers.com` just times out for me

Comment (pajuga) [reply pkg/data/management/machinedriver_data.go:121]: Could you please leave it for now. Yeah that endpoint times out but we have some other that can be needed.

**Review by dramich — APPROVED** [review]

**Comment by dramich** [comment]
@pajuga This pr is for master branch so this would only go out in our next release which is 2.6.0. If this is needed in the 2.5 line then open a backport PR to release/v2.5

**Pull request merged**